dart-impl/tasking/src/internal/dart_tasking_datadeps.c

```diff
@@ -1284,24 +1351,25 @@ dart_tasking_datadeps_match_local_dependency(
                          iter, task, elem, elem->task.local);
           // found one, remove it from the task...
           if (prev == NULL) {
-            task->deps_owned = iter->next_in_task;
+            task->deps_owned = iter->next;
```

Comment:
These changes should already be in `feat-dash-concurrency`. Did something go wrong during merge/rebase?

Reply:
Just rebased and pushed upstream. Your changes actually revert fixes that I added to upstream before.